lib/service/service.go

```diff
@@ -1124,6 +1124,9 @@ func NewTeleport(cfg *servicecfg.Config) (*TeleportProcess, error) {
 		process.initDatabases()
 		serviceStarted = true
 	} else {
+		if process.Config.Discovery.Enabled {
```

Comment:
Should this be `process.Config.Database.Enabled`? The error message you added suggests that.

Reply:
yes 🤦‍♂️  thank you